src/components/ContractInfo.tsx

```diff
@@ -15,15 +18,47 @@ const InfoItem = ({
 );
 
 const ContractInfo = () => {
-  const { GiveStableFee, WantStableFee } =
+  const { GiveStableFee, WantStableFee, MintLimit } =
     useAtomValue(governedParamsAtom) ?? {};
-  const { anchorPoolBalance } = useAtomValue(metricsAtom) ?? {};
+  const { anchorPoolBalance, mintedPoolBalance } =
+    useAtomValue(metricsAtom) ?? {};
   const { displayPercent, displayAmount, displayBrandPetname } =
     useAtomValue(displayFunctionsAtom);
 
   const swapDirection = useAtomValue(swapDirectionAtom);
   const fee =
-    swapDirection === SwapDirection.TO_STABLE ? WantStableFee : GiveStableFee;
+    swapDirection === SwapDirection.TO_MINTED ? WantStableFee : GiveStableFee;
+
+  const stableAvailable = useMemo(
+    () => (
+      <InfoItem>
+        {displayBrandPetname(anchorPoolBalance?.brand)} Available
+        <div className="pr-2">
+          {anchorPoolBalance && displayAmount(anchorPoolBalance)}
+        </div>
+      </InfoItem>
+    ),
+    [anchorPoolBalance, displayAmount, displayBrandPetname]
+  );
+
+  const mintedAvailable = useMemo(
+    () => (
+      <InfoItem>
+        {displayBrandPetname(MintLimit?.brand)} Available
+        <div className="pr-2">
+          {MintLimit &&
```

Comment:
why would `MintLimit` not be defined? seems like an error.

if you're waiting for the params to be read, consider exiting earlier

Reply:
Changed to exit earlier